src/compose/ComposeMenu.js

```diff
@@ -3,9 +3,11 @@ import React, { PureComponent } from 'react';
 import type { ComponentType } from 'react';
 import { Platform, View } from 'react-native';
 import type { DocumentPickerResponse } from 'react-native-document-picker';
-// $FlowFixMe[untyped-import]
-import ImagePicker from 'react-native-image-picker';
+/* $FlowIgnore[cannot-resolve-module] Package only has TS files. Metro
+   transpiles those, and Flow shouldn't care because we wrote a libdef. */
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
```

Comment:
Huh. Does that mean it's not seeing the libdef?

… I think it's not seeing the libdef. If I go munge one of the property names we pass to these functions:
```diff
     launchCamera(
       {
-        mediaType: 'photo',
+        mediaTy: 'photo',
```
I don't get an error.

Perhaps we need some other line in the .flowconfig to tell Flow to ignore the .ts file? And/or to look at the libdef instead. One model might be these lines:
```
[libs]
node_modules/react-native/interface.js
node_modules/react-native/flow/

; If the following files are not explicitly named, Flow ignores them in favor of
; their autogenerated versions in flow-typed/npm. (Contrast commit 6219e5a23,
; where the opposite behavior is exhibited!)
flow-typed/@sentry/react-native_v1.x.x.js
flow-typed/react-native-url-polyfill_vx.x.x.js
flow-typed/expo-screen-orientation_vx.x.x.js

; Explicitly name these so that Flow sees them instead of their
; counterparts in node_modules/react-native:
flow-typed/jest_v26.x.x.js
```

But if with some quick experimentation you don't find a way to get it working, then… we'll ship the version telling Flow to look at the .ts file, I guess, and troubleshoot from there.